### ISSUE TITLE
axe command returns "ENOENT: no such file or directory" error

### ISSUE THREAD

**Issue by CarlyGerard** [dequelabs/axe-core-npm#729]
Product: @axe-core/cli

Expectation: if running <code>axe [url]</code> on the command line (like <code>axe www.github.com</code>), it should start running accessibility checks and return any violations.

Actual: Running <code>axe [url]</code> returns the following error:

<pre><code>
% axe github.com
node:fs:584
  handleErrorFromBinding(ctx);
  ^

Error: ENOENT: no such file or directory, open '/opt/homebrew/lib/node_modules/node_modules/axe-core/axe.js'
    at Object.openSync (node:fs:584:3)
    at Object.readFileSync (node:fs:452:35)
    at getAxeSource (/opt/homebrew/lib/node_modules/@axe-core/cli/dist/cli.js:106:28)
    at Command.cli (/opt/homebrew/lib/node_modules/@axe-core/cli/dist/cli.js:318:20)
    at Command.listener [as _actionHandler] (/opt/homebrew/lib/node_modules/@axe-core/cli/node_modules/commander/lib/command.js:482:17)
    at /opt/homebrew/lib/node_modules/@axe-core/cli/node_modules/commander/lib/command.js:1283:65
    at Command._chainOrCall (/opt/homebrew/lib/node_modules/@axe-core/cli/node_modules/commander/lib/command.js:1177:12)
    at Command._parseCommand (/opt/homebrew/lib/node_modules/@axe-core/cli/node_modules/commander/lib/command.js:1283:27)
    at Command.parse (/opt/homebrew/lib/node_modules/@axe-core/cli/node_modules/commander/lib/command.js:909:10)
    at Object.<anonymous> (/opt/homebrew/lib/node_modules/@axe-core/cli/dist/cli.js:464:9) {
  errno: -2,
  syscall: 'open',
  code: 'ENOENT',
  path: '/opt/homebrew/lib/node_modules/node_modules/axe-core/axe.js'
}
</code></pre>

- axe-core version: 4.7.1
- Node version: 19.4.0 
- Platform: macOS (Monterey, 12.6)
- Chrome version: 113.x

**Comment by Zidious** [comment]
Hey @CarlyGerard, 

Looks like you're using Node19 which is the _current_ but not the _current LTS_ version. Using Node18, do you get the same error?

**Comment by CarlyGerard** [comment]
Thanks @Zidious. I downgraded to node18 as suggested, I still seem to be getting the same error:

<pre><code>
% node -v
v18.16.0
% axe
node:fs:601
  handleErrorFromBinding(ctx);
  ^

Error: ENOENT: no such file or directory, open '/opt/homebrew/lib/node_modules/node_modules/axe-core/axe.js'
    at Object.openSync (node:fs:601:3)
    at Object.readFileSync (node:fs:469:35)
    at getAxeSource (/opt/homebrew/lib/node_modules/@axe-core/cli/dist/cli.js:106:28)
    at Command.cli (/opt/homebrew/lib/node_modules/@axe-core/cli/dist/cli.js:318:20)
    at Command.listener [as _actionHandler] (/opt/homebrew/lib/node_modules/@axe-core/cli/node_modules/commander/lib/command.js:482:17)
    at /opt/homebrew/lib/node_modules/@axe-core/cli/node_modules/commander/lib/command.js:1283:65
    at Command._chainOrCall (/opt/homebrew/lib/node_modules/@axe-core/cli/node_modules/commander/lib/command.js:1177:12)
    at Command._parseCommand (/opt/homebrew/lib/node_modules/@axe-core/cli/node_modules/commander/lib/command.js:1283:27)
    at Command.parse (/opt/homebrew/lib/node_modules/@axe-core/cli/node_modules/commander/lib/command.js:909:10)
    at Object.<anonymous> (/opt/homebrew/lib/node_modules/@axe-core/cli/dist/cli.js:464:9) {
  errno: -2,
  syscall: 'open',
  code: 'ENOENT',
  path: '/opt/homebrew/lib/node_modules/node_modules/axe-core/axe.js'
}

Node.js v18.16.0
</code></pre>

**Comment by Zidious** [comment]
Hey all - sorry for the delay, I have a draft PR open that should land soon we will likely a stable release shortly after it is merged.

**Comment by padmavemulapati** [comment]
On revert ESM , we can overcome this error
Valdiated with the latest canary build for `@axe-core/cli` (4.7.2-alpha.390)
```
MacBook-Pro:cli dq_user$ axe www.google.com --browser chrome
Running axe-core 4.7.2 in chrome
Error: Error: Server was killed with SIGKILL
    at /usr/local/lib/node_modules/@axe-core/cli/node_modules/selenium-webdriver/remote/index.js:260:24
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
Please report the problem to: https://github.com/dequelabs/axe-core-npm/issues/

MacBook-Pro:cli dq_user$ axe www.google.com --browser chrome
Running axe-core 4.7.2 in chrome

Testing http://www.google.com ... please wait, this may take a minute.

  Violation of "aria-allowed-role" with 1 occurrences!
    Ensures role attribute has an appropriate value for the element. Correct invalid elements at:
     - #APjFqb
    For details, see: https://dequeuniversity.com/rules/axe/4.7/aria-allowed-role

  Violation of "aria-valid-attr-value" with 2 occurrences!
    Ensures all ARIA attributes have valid values. Correct invalid elements at:
     - #APjFqb
     - .ayzqOc
    For details, see: https://dequeuniversity.com/rules/axe/4.7/aria-valid-attr-value

  Violation of "color-contrast" with 8 occurrences!
    Ensures the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds. Correct invalid elements at:
     - .uU7dJb
     - .AghGtd > .pHiOh:nth-child(1)
     - .AghGtd > .pHiOh:nth-child(2)
     - .pHiOh:nth-child(3)
     - .pHiOh:nth-child(4)
     - .iTjxkf > .pHiOh:nth-child(1)
     - .iTjxkf > .pHiOh:nth-child(2)
     - .ayzqOc
    For details, see: https://dequeuniversity.com/rules/axe/4.7/color-contrast

  Violation of "region" with 8 occurrences!
    Ensures all page content is contained by landmarks. Correct invalid elements at:
     - .gb_Bd > div:nth-child(1)
     - .gb_oa
     - .lnXdpd
     - #SIvCob
     - .uU7dJb
     - .AghGtd
     - .iTjxkf > .pHiOh:nth-child(1)
     - .iTjxkf > .pHiOh:nth-child(2)
    For details, see: https://dequeuniversity.com/rules/axe/4.7/region

19 Accessibility issues detected.
Testing complete of 1 pages

Please note that only 20% to 50% of all accessibility issues can automatically be detected. 
Manual testing is always required. For more information see:
https://dequeuniversity.com/curriculum/courses/testingmethods

MacBook-Pro:cli dq_user$ axe --version
4.7.2-alpha.390+0d0c9d1
```
